lib/mix/lib/mix/tasks/deps.clean.ex

```diff
@@ -73,6 +73,14 @@ defmodule Mix.Tasks.Deps.Clean do
     apps -- Enum.map(deps, &Atom.to_string(&1.app))
   end
 
+  defp invalid_path_warning?(path, dependency) do
+    unless File.dir?(path) do
+      Mix.shell.error "warning: the dependency #{dependency} is not present in the build directory of depencencies"
```

Comment:
I think the error message looks good, I think we can just remove the trailing "of dependencies". :)